metagpt/actions/action_node.py

```diff
@@ -469,7 +469,7 @@ async def auto_review(self, template: str = REVIEW_TEMPLATE) -> dict[str, str]:
             return dict()
 
         prompt = template.format(
-            nodes_output=json.dumps(nodes_output, ensure_ascii=False, indent=4), tag=TAG, constraint=FORMAT_CONSTRAINT
+            nodes_output=json.dumps(nodes_output, ensure_ascii=False), tag=TAG, constraint=FORMAT_CONSTRAINT
```

Comment:
could you provide more evidence for this?

Reply:
see `other` part in PR desc